src/com/comeet/utilities/Validator.java

```diff
@@ -0,0 +1,15 @@
+package com.comeet.utilities;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+    
+    private static final Pattern VALID_EMAIL_ADDRESS_REGEX = 
```

Comment:
Email addresses are very difficult to validate by regex.  
http://stackoverflow.com/questions/201323/using-a-regular-expression-to-validate-an-email-address
What standard did you use here? Please cite it.